packages/passwordless-auth-construct/src/otp/otp_challenge_service.test.ts

```diff
@@ -50,78 +44,6 @@ void describe('OTP Challenge', () => {
     otpChallenge = new OtpChallengeService(deliveryServiceFactory, otpConfig);
   });
 
-  void describe('validateCreateEvent()', () => {
```

Comment:
*note:* These tests were moved to a new file since this logic was abstracted out for reuse in magic link.